isis/scripts/downloadIsisData

```diff
@@ -110,8 +113,26 @@ def create_rclone_arguments(destination, mission_name, ntransfers=10, rclone_kwa
     if source_type == "naifKernels":
         destination = os.path.join(destination, "kernels")
 
-    extra_args=[f"{mission_name}",f"{destination}", "--progress", f"--checkers={ntransfers}", f"--transfers={ntransfers}", "--track-renames", f"--log-level={log.getLevelName(log.getLogger().getEffectiveLevel())}"]
+    # Add default filter flag
+    filter_args = ["--filter", f"{exclude_string}"]
+
+
+   # Check for additional include and exclude flags
+    include_args = [arg for arg in rclone_kwargs if arg.startswith("--include")]
+    exclude_args = [arg for arg in rclone_kwargs if arg.startswith("--exclude")]
+    filter_args_provided = [arg for arg in rclone_kwargs if arg.startswith("--filter")]
 
+    # Merge any additional include and exclude flags with the default filter flag
+    if filter_args_provided:
+        filter_args = filter_args_provided
+    else:
+        if include_args:
+            filter_args[1] += "," + ",".join([arg.split("=")[1] for arg in include_args]) + "+"
+        if exclude_args:
+            filter_args[1] += "," + ",".join([arg.split("=")[1] for arg in exclude_args])
```

Comment:
Looking at this, it still looks like the flag isn't being set correctly. remember, from the docs I linked, that regexes that need to be includes need to be prefixed with "+" and things that are to be ignored with "-" for every regex. These basically have to be added as there own --filter flag. So you can't use the "{-x,- y, +z }" syntax. e.g `rclone --filter "- gallileo/kernels/ck/prime_mission/*/extended_mission/*" --filter "- /a_older_versions/* ..."`

also, remember if the user passes in a --include, you need to append `--filter "- *"`, in order to replicate the functionality of --include